### PR TITLE
Implement bitwise operator

### PR DESCRIPTION
**Pull request by alitaso345**
Implement and, or, and xor

- [x] and
- [x] or
- [x] xor

### CODE REVIEW

File: lib/wagyu/wasm/compiler.rb [hunk 1]
```diff
@@ -107,6 +107,12 @@ def compile_instr(instr)
         new_var { @stack.pop(2).join(" != ") }
       when :eqz
         new_var { "#{@stack.pop} == 0" }
+      when :and
+        new_var { @stack.pop(2).join(" && ") }
+      when :or
+        new_var { @stack.pop(2).join(" || ") }
```
Comment (edvakf) [inline lib/wagyu/wasm/compiler.rb]: Actually, "&&" and "||" have to be "&" and "|" respectively. 
https://medium.com/rubycademy/ruby-bitwise-operators-da57763fa368

File: test/wasm_test.rb [hunk 2]
```diff
@@ -13,6 +13,30 @@ def test_square
     assert_equal(9, result)
   end
 
+  def test_and
+    instance = instantiate("and.wasm")
+    assert_equal(false, instance.exports.and(false, false))
+    assert_equal(false, instance.exports.and(false, true))
+    assert_equal(false, instance.exports.and(true, false))
+    assert_equal(true, instance.exports.and(true, true))
```
Comment (edvakf) [inline test/wasm_test.rb]: The values passed need to be i32 values because "and", "or" and "xor" implement i32 operations.

Comment (alitaso345) [reply test/wasm_test.rb]: @edvakf 9c8a4a8 fixed!

File: lib/wagyu/wasm/compiler.rb (hunk 1 above)
Comment (alitaso345) [reply lib/wagyu/wasm/compiler.rb]: @edvakf fixed! d8bb610

**Pull request merged**